admin/app/assets/javascripts/jquery_ujs/insert.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Add data-insert to an element. When that element is also data-remote, the
+ * server's response will be inserted into the element matching the value of
+ * the data-insert attribute. (with WORKAREA modules initialized).
+ */
+
+$(document).on('ajax:success', function (event, response) {
+    var $element = $(event.target),
+        $target = $($element.data('insert')),
+        $content = $(response);
+
+    if (_.isEmpty($target)) { return; }
+    if (! $element.is('[data-insert]')) { return; }
+
+    if ($element.is('[data-insert-after]')) {
+        $content.insertAfter($target);
+    } else {
+        $target.html($content);
```

Comment:
Since this is called "insert", it seems a little weird to use `html` since that will replace the content of `$target`. Did you mean `append`?

Reply:
I didn't, but I also realized i don't need the `html` behavior in product bundles anymore. So I just update the module